e2e/test/scenarios/permissions/sandboxing/sandboxing-via-ui.cy.spec.ts

```diff
@@ -44,9 +58,18 @@ describe(
             (item: { model: string }) => item.model !== "dashboard",
           ),
         };
+
+        cy.log(
+          "We additionally want to ensure that sandboxed users see filtered results even if the unsandboxed results are cached. So let's cache the unsandboxed results",
+        );
+        cy.request("PUT", "/api/cache", simpleCacheConfiguration).then(() => {
+          cy.log("Populate the caches");
+          getCardResponses(items);
+        });
```

Comment:
my concern here is that we're now _only_ testing that cached results are sandboxed. can we find a way to test some cases of both without doubling the tests?

Reply:
had another idea. we could create 2 snapshots during setup, one uncached, and one cached, then we could use the uncached one at first, and then at the end of each test, restore the cached one and recheck the card responses